run.sh

```diff
@@ -80,18 +78,18 @@ pid=$!
 count=0
 wid=""
 
-while [ -z "$wid" ]; do
+while [[ -z "$HEADLESS" && -z "$wid" ]]; do
     wid=$(wmctrl -l | grep " Google Chrome" | cut -f 1 -d ' ')
     if [ -n "$wid" ]; then
         echo "Chrome Found"
         break
     fi
     sleep 0.5
-    count=$[$count + 1]
+    count=$((count+1))
     echo "Chrome Not Found"
     if [ $count -eq 6 ]; then
         echo "Restarting process"
-        kill $(ps -ef | grep "/chrome/chrome --no-def" | awk '{ print $2 }')
+        kill "$(ps -ef | grep "/chrome/chrome --no-def" | awk '{ print $2 }')"
```

Comment:
shellcheck report for line:
`SC2009: Consider using pgrep instead of grepping ps output`

Reply:
Switched to pkill, thanks